styleguide/source/_patterns/02-organisms/06-topic/02-topic-tools/_topic-tools.scss

```diff
@@ -1,9 +1,10 @@
 .layout.topic_tools {
-  flex-direction: column;
+  @include grid--direction-col();
 }
 
 // The tools block.
 .topic_tools {
```

Comment:
Suggestion to add either half or full gutter (whatever looks best) at the bottom of this element